CI fails: 403 when retrieving repo brownie-mix/github-actions 'API rate limit exceeded'

### The error

I just observed the following, on code that should have passed. (I'd just merged a PR, in the branch the tests passed.)

```
Successfully installed Pillow-9.3.0 PyJWT-1.7.1 ... zipp-3.11.0
Brownie v1.19.2 - Python development framework for Ethereum

  File "brownie/_cli/__main__.py", line 64, in main
    importlib.import_module(f"brownie._cli.{cmd}").main()
  File "brownie/_cli/bake.py", line 26, in main
    path = project.from_brownie_mix(args["<mix>"], args["<path>"], args["--force"])
  File "brownie/project/main.py", line 629, in from_brownie_mix
    default_branch = _get_mix_default_branch(project_name, headers)
  File "brownie/project/main.py", line 1095, in _get_mix_default_branch
    raise ConnectionError(msg)
ConnectionError: Status 403 when retrieving repo brownie-mix/github-actions information from GHAPI: 'API rate limit exceeded for 20.114.68.50. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)'

Missing or forbidden.
If this issue persists, generate a Github API token and store it as the environment variable `GITHUB_TOKEN`:
https://github.blog/2013-05-16-personal-api-tokens/
Error: Process completed with exit code 1.
```

### Discussion

This is probably from commit https://github.com/oceanprotocol/ocean.py/commit/2b49b5649bf616e342e1e28dcd8b73af4bd6a6e2 /  PR https://github.com/oceanprotocol/ocean.py/pull/1167 / issue https://github.com/oceanprotocol/ocean.py/issues/1156.

That commit added `brownie bake github-actions`, which was the cleanest solution to the problem: get `~/.brownie/network-config.yaml` for CI, such that it will stay up-to-date with changes to brownie. (And without adding a large copy-and-pasted file from brownie).

Looks like that call has rate-limiting.

### Candidate solutions

1. Do an authenticated request. ("But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details")
2. Copy and paste file
3. other?

I recommend trying (1) first. It's probably just adding another argument to the `brownie-bake` call that specifies the github account.

--- a/.github/workflows/pytest.yml
+++ b/.github/workflows/pytest.yml
@@ -35,6 +35,8 @@ jobs:
           ls -la "$HOME/.ocean/ocean-contracts/artifacts/"
       - name: Install dependencies
         working-directory: ${{ github.workspace }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements_dev.txt

